models.py

```diff
@@ -1401,3 +1404,40 @@ def __init__(self, *args, **kwargs):
 			nonlinearity = ('leaky_relu', 0.01),
 			**kwargs
 		)
+
+
+class OnnxWrapper():
+	def __init__(self, onnx_model_path, providers=['CUDAExecutionProvider', 'CPUExecutionProvider']):
+		self.dict = dict
+		self.bpe_only = False
+		self.onnxruntime_session = onnxruntime.InferenceSession(onnx_model_path)
+
+		if providers:
+			self.onnxruntime_session.set_providers(list(providers))
```

Comment:
is `list` call necessary? can't you use `providers` directly?

Reply:
done